src/lz4frame.jl

```diff
@@ -65,13 +65,22 @@ const LZ4F_blockChecksumEnabled = (UInt32)(1)
 const LZ4F_frame = (UInt32)(0)
 const LZ4F_skippableFrame = (UInt32)(1)
 
+@enum BlockSizeID::Cuint default_size = 0 max64KB = 4 max256KB = 5 max1MB = 6 max4MB = 7
+@enum BlockMode::Cuint block_linked = 0 block_independent = 1
+@enum FrameType::Cuint normal_frame = 0 skippable_frame = 1
+
 struct LZ4F_compressOptions_t
     stableSrc::Cuint
     reserved::NTuple{3, Cuint}
 end
 
+struct LZ4F_decompressOptions_t
+    stableDst::Cuint
+    reserved::NTuple{3, Cuint}
+end
+
 struct LZ4F_frameInfo_t
-    blockSizeID::UInt32
+    blockSizeID::Cuint
```

Comment:
Should this be `blockSizeID::BlockSizeID`? Also you should try to stick to underscore separated variable names unless you have a reason.

Reply:
These structs correspond directly to the C code(hence the variable names). I left `blockSizeID` as a `Cuint` to make it easier to debug since if someone messes with the enum type, it'll probably cause the code to misbehave.

Reply:
Sounds good. Just wanted to make sure that was what was going on